src/main/java/com/autotune/analyzer/services/BulkService.java

```diff
@@ -75,33 +150,70 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             String includeParams = req.getParameter("include");
             String excludeParams = req.getParameter("exclude");
             String experiment_name = req.getParameter("experiment_name");
-
-            LOGGER.debug(" include fields: {}", includeParams);
-            LOGGER.debug(" exclude fields: {}", excludeParams);
-
             // Parse the include and exclude parameters into lists
             Set<String> includeFields = includeParams != null ? new HashSet<>(Arrays.asList(includeParams.split(","))) : new HashSet<>(Arrays.asList("summary"));
             Set<String> excludeFields = excludeParams != null ? new HashSet<>(Arrays.asList(excludeParams.split(","))) : Collections.emptySet();
 
-            LOGGER.debug(" include fields: {}", includeFields);
-            LOGGER.debug(" exclude fields: {}", excludeFields);
-            LOGGER.debug(" experiment_name: {}", experiment_name);
-
             // If the parameter is not provided (null), default it to false
             boolean verbose = verboseParam != null && Boolean.parseBoolean(verboseParam);
             BulkJobStatus jobDetails;
-            LOGGER.info("Job ID: " + jobID);
-            if (jobStatusMap.isEmpty()) {
-                sendErrorResponse(
-                        resp,
-                        null,
-                        HttpServletResponse.SC_NOT_FOUND,
-                        JOB_NOT_FOUND_MSG
-                );
-                return;
+            // By default cache_job_in_mem set to false , If set it to true it will store in memory
+            if (KruizeDeploymentInfo.cache_job_in_mem) {
```

Comment:
This should be renamed as `TEST_USE_ONLY_cache_job_in_mem` or something like that so that this is not used for production purposes

Reply:
done